packages/node_modules/@ciscospark/plugin-phone/src/call.js

```diff
@@ -949,6 +949,9 @@ const Call = SparkPlugin.extend({
   },
 
   _updateMedia(payload) {
+    if (payload.sdp && !payload.sdp.includes(`b=`)) {
```

Comment:
no, wait, wrong again. both audio and video should have b lines. misread some code.